exercises/book-store/canonical-data.json

```diff
@@ -0,0 +1,101 @@
+{
+   "book-store": {
+      "description": [
+                       "Calculate lowest price for shopping basket only ",
+                       "containing books from a single series"
+                     ],
+      "cases": [
+         {
+            "description": "A basket containing only a single book.",
+            "basket": [1],
+            "targetgrouping": [1],
+            "expected": 8.0
+         },
+         {
+            "description":  "A basket containing only two of the same book.",
+            "basket": [2, 2],
+            "targetgrouping": [[2], [2]],
+            "expected": 16.0
+         },
+         {
+            "description": "No charge to carry around an empty basket.",
+            "basket": [],
+            "targetgrouping": [],
+            "expected": 0.0
+         },
+         {
+            "description": "A basket containing only two different books.",
+            "basket": [1, 2],
+            "targetgrouping": [1, 2],
+            "expected": 15.2
+         },
+         {
+            "description": "A basket of three different books.",
+            "basket": [1, 2, 3],
+            "targetgrouping": [1, 2, 3],
+            "expected": 21.60
+         },
+         {
+            "description": "A basket of four different books.",
+            "basket": [1, 2, 3, 4],
+            "targetgrouping": [1, 2, 3, 4],
+            "expected": 25.60
+         },
+         {
+            "description": "A basket of five different books.",
+            "basket": [1, 2, 3, 4, 5],
+            "targetgrouping": [1, 2, 3, 4, 5],
+            "expected": 30.00
+         },
+         {
+            "description": [
+                            "A basket containing eight books consisting of a ",
```

Comment:
Any way to make it clear either in the description or elsewhere that we test this case to make sure that the student finds `[[1, 2, 3, 4], [1, 2, 3, 5]]` rather than `[[1, 2, 3, 4, 5], [1, 2, 3]]`?

It's interesting to me because I think this is the only case in this JSON file where the greedy solution won't work, so it's particularly important to me that the description point that out.